hack/tools/release/notes.go

```diff
@@ -231,14 +232,15 @@ func run() int {
 		merges[superseded] = append(merges[superseded], "- `<insert superseded bumps and reverts here>`")
 	}
 
+	fmt.Println("<!-- markdownlint-disable no-inline-html , line-length -->\n")
```

Comment:
```suggestion
	fmt.Println("<!-- markdownlint-disable no-inline-html line-length -->\n")
```
The syntax is with spaces, sorry my bad first time around.

Reply:
fixed